docs/latest/design/accesslogging.md

```diff
@@ -0,0 +1,251 @@
+# Observability: Access Logging
+
+## Overview
+
+Envoy supports extensible access logging to different sinks, File, gRPC etc.
+Envoy supports customizable access log formats using predefined fields as well as arbitrary HTTP request and response headers.
+Envoy supports several built-in access log filters and extension filters that are registered at runtime.
+
+Envoy Gateway leverages [Gateway API][] for configuring managed Envoy proxies. Gateway API defines core, extended, and
+implementation-specific API [support levels][] for implementors such as Envoy Gateway to expose features. 
+Since access logging is not covered by `Core` or `Extended` APIs, EG should provide an easy to config access log formats and sinks per `EnvoyProxy`.
+
+## Goals
+
+- Support send access logging to `File` or `OpenTelemetry` backend
+- Support access log filters base on [CEL][] expression
```

Comment:
not a strong preference, would be prefer if `filtering` was added in a follow PR to keep this design small

Reply:
ok, I can remove it first.

Reply:
I tend to agree with this approach – log everything first, then support filtering.